src/main/scala/io/viash/config/arguments/FileArgument.scala

```diff
@@ -47,6 +47,18 @@ case class FileArgument(
   @default("Empty")
   alternatives: OneOrMore[String] = Nil,
 
+  @description("A clean notation of the argument's name. This is used when generating documentation.")
+  @example("label: an argument", "yaml")
+  @default("Empty")
+  @since("Viash 0.9.0")
+  label: Option[String] = None,
+
+  @description("A short summary of the argument. This is used when generating documentation.")
+  @example("summary: Use this argument to do X", "yaml")
+  @default("Empty")
+  @since("Viash 0.9.0")
+  summary: Option[String] = None,
+
   @description("A description of the argument. This will be displayed with `--help`.")
```

Comment:
```suggestion
  @description("A clean version of the argument's name. This is only used for documentation.")
  @example("label: \"My argument\"", "yaml")
  @default("Empty")
  @since("Viash 0.9.0")
  label: Option[String] = None,

  @description("A one-sentence summary of the argument. This is only used for documentation.")
  @example("summary: \"This argument sets XYZ.\"", "yaml")
  @default("Empty")
  @since("Viash 0.9.0")
  summary: Option[String] = None,

  @description("A description of the argument. This is only used for documentation. Multiline descriptions are supported.")
  @example(
    """description: |
      |  A (multiline) description of the purpose of
      |  this argument.""".stripMargin, "yaml")
  @default("Empty")